CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Giacomo Pozzoni [jackpoz]
 - [blarrywangmsft]
 - Koen Zomers [koenzomers]
+- Christian Veenhuis [ChVeen]
```

Comment:
Please undo this change.
This file is to be modified by maintainers only.

Reply:
For my last PR someone had to add this entry for me and I thought I simply forget this.
Removed :-)